hwdb.d/20-dmi-id.hwdb

```diff
@@ -23,6 +23,14 @@ dmi:*:pnTobefilledbyO.E.M.:*
 dmi:*:pnToBeFilledByO.E.M.:*
     ID_PRODUCT_NAME_IS_RUBBISH=1
 
+dmi:*:catN/A:*
```

Comment:
Please also add `Defaultstring`.

Maybe also add `Default string` for this and the previous rubbish filters, as that is reported on my topton mini pc.